src/cleanupEquation.js

```diff
@@ -0,0 +1,20 @@
+import closeParens from "./closeParens";
+import addMultiplier from "./addMultiplier";
+
+const getLastChar = (from) => {
+  return from.slice(-1);
+};
+
+const cleanupEquation = (result) => {
+  var tempResult = result;
```

Comment:
I can see how creating `var tempResult` makes for a cleaner refactor. Good job! I'm noticing the `var tempResult` wasn't removed from App.js, why? Also, now that it's extracted though, do we need to keep it?

Reply:
This is an old version of the code...the latest version I pushed with failing tests does not have this code...

Reply:
In order to understand why your current commit is failing, we have to go through all your commits. I'm making notes as I do.

Reply:
Oh okay...